_docs/Popup/popup.md

```diff
@@ -0,0 +1,42 @@
+# Popup Component
+
+**Popup** is a component that pops up content and positions it next to an anchor element. A Popup is overlayed over the page. A Popup is a temporary view and only one popup can be rendered on the page at one time.
```

Comment:
@idoros I had asked about this and was informed that only one popup at a time can be rendered.